src/app/fdctl/run/tiles/fd_quic.c

```diff
@@ -648,7 +569,9 @@ unprivileged_init( fd_topo_t *      topo,
   FD_SCRATCH_ALLOC_INIT( l, scratch );
   fd_quic_ctx_t * ctx = FD_SCRATCH_ALLOC_APPEND( l, alignof( fd_quic_ctx_t ), sizeof( fd_quic_ctx_t ) );
   FD_SCRATCH_ALLOC_APPEND( l, fd_ip_align(), fd_ip_footprint( 256UL, 256UL ) );
-  FD_SCRATCH_ALLOC_APPEND( l, fd_alloc_align(), fd_alloc_footprint() );
+
+  fd_quic_limits_t limits = quic_limits( tile );
+  void * quic_mem = FD_SCRATCH_ALLOC_APPEND( l, fd_quic_align(), fd_quic_footprint( &limits ) );
```

Comment:
Not valid to move this here. It needs to be after SCRATCH_ALLOC( aio ) (must be same ordering as the footprint func)